examples/xeval.ts

```diff
@@ -33,7 +33,7 @@ OPTIONS:
 ARGS:
   <code>`;
 
-export type XevalFunc = (v: string) => void;
+export type XevalFunc = (v: string) => unknown | Promise<unknown>;
```

Comment:
It looks like `xEvalFunc` is an async function here. Won't that make the return type only be `Promise<unknown>`?

Reply:
It makes sense. 👍